x-pack/plugins/stack_connectors/public/connector_types/cases_webhook/validator.ts

```diff
@@ -81,16 +81,18 @@ export const containsCommentsOrEmpty =
   (message: string) =>
   (...args: Parameters<ValidationFunc>): ReturnType<ValidationFunc<any, ERROR_CODE>> => {
     const [{ value, path }] = args;
+
+    if (value === null || value === undefined || value === '') {
```

Comment:
I can that **maybe because of the validation here** I can save the form when `Create comment URL` is missing.

<img width="719" alt="Screenshot 2024-11-07 at 12 16 42" src="https://github.com/user-attachments/assets/2a0c19d9-bcde-45cf-acde-f0cc8d88fc01">

It would be nice if we improved the validation to guarantee that if the user defines the URL, he also needs to have the body and the method. These 3 belong together.